ddmrp_production_equivalent/models/mrp_production.py

```diff
@@ -36,8 +36,12 @@ def _generate_raw_move(self, bom_line, line_data):
         product_equivalent = self._get_product_equivalent(bom_line,
                                                           line_data['qty'])
         if product_equivalent:
+            body = _('%s has been replaced by %s.' %
+                     sm.product_id.name_get()[0][1],
+                     product_equivalent.name_get()[0][1])
```

Comment:
Tried to run this code and got this error.
*** TypeError: not enough arguments for format string

Would suggest the following lines

            body = _('%s has been replaced by %s.' %
                     (sm.product_id.name_get()[0][1],
                     product_equivalent.name_get()[0][1]))